docker-compose-frontend-tests.yml

```diff
@@ -0,0 +1,67 @@
+# Docker Compose file for Malwarecage development environment
+
+version: "3.3"
+services:
+  mwdb:
+    build:
+      context: .
+      dockerfile: deploy/docker/Dockerfile
+    depends_on:
+      - postgres
+      - redis
+    restart: on-failure
+    env_file:
+      # NOTE: use gen_vars.sh in order to generate this file
+      - mwdb-vars.env
+    environment:
+      UWSGI_PY_AUTORELOAD: 1
+      UWSGI_ENABLE_THREADS: 1
+      MALWARECAGE_MAIL_SMTP: "mailhog:1025"
+      MALWARECAGE_MAIL_FROM: "noreply@malwarecage.dev"
+      MALWARECAGE_RECAPTCHA_SITE_KEY: "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+      MALWARECAGE_RECAPTCHA_SECRET: "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+      MALWARECAGE_ENABLE_REGISTRATION: 1
+    volumes:
+      - "./core:/app/core"
+      - "./migrations/versions:/app/migrations/versions"
+      - "./mail_templates:/app/mail_templates"
+      - "./model:/app/model"
+      - "./plugins:/app/plugins"
+      - "./resources:/app/resources"
+      - "./app.py:/app/app.py"
+      - "./plugin_engine.py:/app/plugin_engine.py"
+  malwarefront:
+    depends_on:
+      - mwdb
+    build:
+      context: .
+      dockerfile: deploy/docker/Dockerfile-web-dev
+    ports:
+      - "80:3000"
+    environment:
+      HOST: 0.0.0.0
+      REACT_APP_API_URL: "/api/"
+    volumes:
+      - "./malwarefront/public:/app/public"
+      - "./malwarefront/src:/app/src"
+      - "./plugins:/plugins"
+    restart: on-failure
+  postgres:
+    image: postgres
+    restart: always
+    env_file:
+      # NOTE: use gen_vars.sh in order to generate this file
+      - postgres-vars.env
+  redis:
+    image: redis:alpine
+  mailhog:
+    image: mailhog/mailhog:latest
+    ports:
+      - "127.0.0.1:8025:8025"
+  frontend-tests:
+    build:
+      context: .
+      dockerfile:  tests-frontend/Dockerfile
+    depends_on:
+      - mwdb
+      - malwarefront
```

Comment:
We already have Docker Compose file for e2e tests.

`frontend-tests` should be moved to `docker-compose-e2e.yml` and whole file `docker-compose-frontend-tests.yml` should be deleted.